tests/integration/test_cases/milvus_vdb_auth.py

```
@@ -202,14 +202,15 @@ async def _test_grant_read_and_verify(self) -> bool:
-            for priv in ("Query", "Search", "DescribeCollection", "Load"):
+            for priv in ("Query", "Search", "DescribeCollection", "Load", "GetLoadState", "GetLoadingProgress"):
                 try:
-                    if priv == "DescribeCollection":
+                    # DescribeCollection, GetLoadState, and GetLoadingProgress are Global privileges
+                    if priv in ("DescribeCollection", "GetLoadState", "GetLoadingProgress"):
                         _grant_collection_privilege(client, self.reader_role, "Global", self.collection_name, privilege=priv)
                     else:
                         _grant_collection_privilege(client, self.reader_role, "Collection", self.collection_name, privilege=priv)
-                except Exception:
-                    pass
+                except Exception as e:
```

Ruff (0.14.7)

Do not catch blind exception: `Exception`

(BLE001)